### ISSUE TITLE
Affinity Matrix

### ISSUE THREAD

**Issue by Hfhstrive** [xulianuwa/AuxSegNet#4]
Thanks for your code sharing, I have a question：
In non_local.py  line 33: 
 f_div_C = F.softmax(f, dim=-1)
Why use softmax here, and what does dim=-1 mean.
I want to visualize the affinity matrix, how to do it.
